hazelcast/src/main/java/com/hazelcast/spi/properties/HazelcastProperties.java

```diff
@@ -234,4 +202,49 @@ public int getSeconds(HazelcastProperty property) {
         throw new IllegalArgumentException(format("value '%s' for property '%s' is not a valid %s value",
                 value, property.getName(), enumClazz.getName()));
     }
+
+    /**
+     * Lookup a {@link HazelcastProperty}'s {@link String} value in the given {@code properties} or in
+     * {@link System#getProperties()}, as specified in this class' javadoc.
+     * @param properties    user-provided {@link Properties}. If {@code null}, a {@link NullPointerException} will
+     *                      be thrown.
+     * @param property      the {@link HazelcastProperty} whose value is looked up
+     * @return              the configured value
+     */
+    public static String getString(Properties properties, HazelcastProperty property) {
```

Comment:
The property lookup implementation was extracted to this `public static` method, so that properties can be looked up from contexts where `Node` is not available.